Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md

```diff
@@ -1,34 +1,34 @@
 
 # Automatic Machine Learning Introduction with Driverless AI
 
-## Outline
+## Contorno
 
-- [Objective](#objective)
-- [Prerequisites](#prerequisites)
-- [Task 1: Product Tour](#task-1-product-tour)
-- [Task 2: Automatic Machine Learning Concepts](#task-2-automatic-machine-learning-concepts)
+- [Objetivo](#objective)
+- [Requisitos Antes de Comenzar](#prerequisites)
+- [Tarea 1: Tour del Producto](#task-1-product-tour)
+- [Tarea 2: Conceptos de Machine Learning(Aprendizaje de maquina)](#task-2-automatic-machine-learning-concepts)
 - [Task 3: Load Data](#task-3-load-data)
-- [Task 4: Explore Data Details and AutoViz](#task-4-explore-data-details-and-autoviz)
-- [Task 5: Launch First Experiment](#task-5-launch-first-experiment)
-- [Task 6: Explore Feature Engineering](#task-6-explore-feature-engineering)
-- [Task 7: Explore Experiment Results](#task-7-explore-experiment-results)
-- [Task 8: MLI Report for Non-Time-Series](#task-8-mli-report-for-non-time-series)
-- [Task 9: Experiment Summary and Autoreport](#task-9-experiment-summary-and-autoreport)
-- [Next Steps](#next-steps)
-- [Appendix: Project Workspace](#appendix-project-workspace)
+- [Tarea 4: Explore los Detalles de Datos y AutoViz](#task-4-explore-data-details-and-autoviz)
+- [Tarea 5: Lanzar primer experimento](#task-5-launch-first-experiment)
+- [Tarea 6: Explora la Ingeniería de Características](#task-6-explore-feature-engineering)
+- [Tarea 7: Explora Resultados del Experimento](#task-7-explore-experiment-results)
+- [Tarea 8: Informe MLI para series no cronológicas](#task-8-mli-report-for-non-time-series)
+- [Tarea 9: Resumen del experimento y autoinforme](#task-9-experiment-summary-and-autoreport)
+- [Próximos pasos](#next-steps)
+- [Apéndice: Espacio de trabajo del proyecto](#appendix-project-workspace)
 
 
-## Objective
+## Objetivo
 
-For this tutorial, we will explore the Titanic dataset from the perspective of a passenger life insurance company with [H2O.ai's](https://www.h2o.ai/) enterprise product, [Driverless AI](https://www.h2o.ai/products/h2o-driverless-ai/). We will explore possible riks factors derived from this dataset that could have been considered when selling passenger insurance during this time. More specifically, we will create a predictive model to determine what factors contributed to a passenger surviving.
+Para este tutorial, exploraremos el conjunto de datos Titanic desde la perspectiva de una compañía de seguros de vida de pasajeros con el producto empresarial [H2O.ai's](https://www.h2o.ai/), [Driverless AI (AI sin conductor)](https://www.h2o.ai/products/h2o-driverless-ai/). Exploraremos posibles factores de riesgo derivados de este conjunto de datos que podrían haberse considerado al vender el seguro de pasajeros durante este tiempo. Más específicamente, crearemos un modelo predictivo para determinar qué factores contribuyeron a la supervivencia de un pasajero.
```

Comment:
Suggestion to change: 
Para este tutorial, exploraremos el conjunto de datos Titanic desde la perspectiva de una compañía de seguros de vida de pasajeros con el producto empresarial [H2O.ai's](https://www.h2o.ai/), [Driverless AI (AI sin conductor)](https://www.h2o.ai/products/h2o-driverless-ai/).

To:
Para este tutorial, exploraremos el conjunto de datos Titanic desde la perspectiva de una compañía de seguros de vida de pasajeros con el producto empresarial [Driverless AI (AI sin conductor)](https://www.h2o.ai/products/h2o-driverless-ai/) de [H2O.ai's](https://www.h2o.ai/).